app/assets/stylesheets/active_admin.scss

```diff
@@ -5,9 +5,6 @@ $table-stripe-color: #f5f5f5;
 @import "active_admin/mixins";
 @import "active_admin/base";
 @import "select2";
-@import "bootstrap-sprockets";
-@import "bootstrap/variables";
-@import "bootstrap/glyphicons";
```

Comment:
We should keep the icons working on the admin side, see:

<img width="440" alt="Captura de pantalla 2025-03-04 a las 13 42 38" src="https://github.com/user-attachments/assets/c117fd0b-92b9-4099-bfa6-0aec5b80695b" />